files/templates/alertmanager/alertmanager.yaml

```diff
@@ -63,7 +63,9 @@ route:
   - receiver: team_atlas_slack
     matchers:
     [[- if eq .Pipeline "stable" ]]
-    - severity!="page"
+    - severity="notify"
```

Comment:
You describe the opposite in the PR: severity=page

Reply:
I agree this deserve a comment as the logic is twisted here.

We sent only `notify` alert to slack for **stable** installation, `page` alerts goes into opsgenie.
For other installations (testing) we send notify and page alert to slack, as we do not see them in opsgenie.